application/menunode.class.inc.php

```diff
@@ -746,7 +757,7 @@ public function GetEntriesCount()
 	}
 
 	/**
-	 * @return string
+	 * @return string The "+" dictionary entry for this menu if exists, otherwise the Title
```

Comment:
Actually this isn't totally true as if there aren't any "+" dict key, we won't simply call GetTitle...

```suggestion
	 * @return string The "+" dictionary entry for this menu if exists, otherwise the Title (if we have a parent title, will output parentTitle / currentTitle)
```

Reply:
Do you have an example, where this combination of title is occurring, I have never seen it on any iTop...
![sub-menu-title](https://user-images.githubusercontent.com/42336698/209327684-5093691e-e8c8-4b6f-b324-8c57f26c2d52.png)

Reply:
I described the code I saw in GetLabel(), but currently we are using GetTitle in the menu right ? 
There are multiple callers for this \MenuNode::GetLabel method but might not be very easy to spot where the parent + current string can appear ?